lib/cloud/storage/gridfsStorage.js

```diff
@@ -0,0 +1,101 @@
+var q = require('q');
+var fs = require('fs');
+var mongo = require('mongodb');
+var Grid = require('gridfs-stream');
+var MongoClient = require('mongodb').MongoClient;
+
+var gfsPromise;
+
+/**
+ * Storage engine that utilizes gridfs mongodb connection to store files
+ */
+var gridFsStorage = {
+  init: init,
+  writeFile: writeFile,
+  streamFile: streamFile
+};
+
+/**
+ * Init module
+ *
+ * @param storageConfiguration configuration that should be passed from client
+ *
+ * var storageConfig = {
```

Comment:
Update comment to have the right config params as per the documentation: 
`var storageConfig = {
   gridFs: {
     mongoUrl: "mongodb://localhost:27017/files"
   }
 };`